chapters/statemachines.tex

```diff
@@ -428,33 +430,19 @@ \subsection{Merging Connections to Multiple Outputs}\label{merging-connections-t
 \begin{nonnormative}
 Since instances of blocks can be used as states of a state
 machine it is natural to extend the connection semantics of Modelica to
-allow several outputs to be connected to one input.
+allow several outputs to be connected, viewing the combination of them as just one source of the signal.
 \end{nonnormative}
 
-It is possible to connect several outputs to an input if all the outputs
-come from states of the same state machine. In such cases, we get the
-following constraint equations:
+It is possible to connect outputs each coming from different states of state machines together - and connect this with other causal connectors.
+These outputs are combined seen as one source of the signal, and give the following constraint equations,
 \begin{lstlisting}[language=modelica,escapechar=!]
 u!\textsubscript{1}! = u!\textsubscript{2}! = ... = y!\textsubscript{1}! = y!\textsubscript{2}! = ...
 \end{lstlisting}
-with u\textsubscript{i} inputs and y\textsubscript{i} outputs. The
-semantics is defined as follows. Introduce a variable v representing the
-signal flow and rewrite the equation above as a set of equations for
-u\textsubscript{i} and a set of assignment equations for v:
-\begin{lstlisting}[language=modelica,escapechar=!]
-v := if activeState(state!\textsubscript{1}!) then y!\textsubscript{1}! else last(v);
-v := if activeState(state!\textsubscript{2}!) then y!\textsubscript{2}! else last(v);
-...
-u!\textsubscript{1}! = v
-u!\textsubscript{2}! = v
-...
-\end{lstlisting}
-
-The merge of the definitions of v is then made according to \cref{merging-variable-definitions}:
-Merging Variable Definitions. The result is after
-simplification:
+with y\textsubscript{i} outputs from different states of the state-machine and u\textsubscript{i} other causal variables. The
```

Comment:
No I wanted to avoid "input", as you could also connect all of the inside outputs from different states of the state-machine to one outside output in the class everything is located in (or to an inside input).

Reply:
Is see.  Resolving.